site/catalog/toggles.js

```diff
@@ -2,27 +2,21 @@ import React from 'react';
 
 const colors = [
   null,
-  'gray-faint',
-  'gray-light',
   'gray',
-  'gray-dark',
```

Comment:
Curious about all these `gray-{varient}` omissions? They ... still exist but you're removing them from the catalog?